<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>reconstruction_window</class>
 <widget class="QMainWindow" name="reconstruction_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>430</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>550</width>
    <height>430</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Arial</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Reconstruction</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="margin">
     <number>0</number>
    </property>
    <item>
     <widget class="QToolBox" name="toolBox">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <property name="tabSpacing">
       <number>0</number>
      </property>
      <widget class="QWidget" name="page_3">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>422</width>
         <height>146</height>
        </rect>
       </property>
       <attribute name="label">
        <string>Step 1: setup brain mask</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout">
              <property name="spacing">
               <number>0</number>
              </property>
              <item>
               <widget class="QToolButton" name="load_mask">
                <property name="text">
                 <string>Open...</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QToolButton" name="save_mask">
                <property name="text">
                 <string>Save...</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QToolButton" name="thresholding">
                <property name="text">
                 <string>Thresholding</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QToolButton" name="dilation">
                <property name="text">
                 <string>Dilation</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QToolButton" name="erosion">
                <property name="text">
                 <string>Erosion</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QToolButton" name="smoothing">
                <property name="text">
                 <string>Smoothing</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QToolButton" name="defragment">
                <property name="text">
                 <string>Defragment</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer_2">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QGraphicsView" name="graphicsView"/>
            </item>
            <item>
             <widget class="QSlider" name="SlicePos">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>533</width>
         <height>444</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <attribute name="label">
        <string>Step 2: select reconstruction method</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string>Reconstruction Method</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QRadioButton" name="DTI">
             <property name="text">
              <string>DTI</string>
             </property>
             <property name="checked">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="DSI">
             <property name="text">
              <string>DSI</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="QBI">
             <property name="text">
              <string>QBI</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="GQI">
             <property name="text">
              <string>GQI</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="QDif">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="text">
              <string>Q-Space Diffeomorphic</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="QSDRT">
             <property name="text">
              <string>QSDR Template</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="ResolutionBox">
          <property name="title">
           <string/>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_8">
           <item>
            <widget class="QLabel" name="label_8">
             <property name="text">
              <string>Output Resolution (high resolution requires large memory space)</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDoubleSpinBox" name="mni_resolution">
             <property name="maximumSize">
              <size>
               <width>75</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="minimum">
              <double>0.500000000000000</double>
             </property>
             <property name="maximum">
              <double>3.000000000000000</double>
             </property>
             <property name="singleStep">
              <double>0.100000000000000</double>
             </property>
             <property name="value">
              <double>2.000000000000000</double>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="DSIOption_2">
          <layout class="QHBoxLayout" name="DSIOption">
           <item>
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>DSI Hamming filter</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="hamming_filter">
             <property name="maximumSize">
              <size>
               <width>75</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="minimum">
              <number>8</number>
             </property>
             <property name="maximum">
              <number>20</number>
             </property>
             <property name="value">
              <number>16</number>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="QBIOption_2">
          <layout class="QHBoxLayout" name="QBIOption">
           <item>
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>QBI Regularization</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDoubleSpinBox" name="regularization_param">
             <property name="maximumSize">
              <size>
               <width>75</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="decimals">
              <number>3</number>
             </property>
             <property name="minimum">
              <double>0.001000000000000</double>
             </property>
             <property name="maximum">
              <double>1.000000000000000</double>
             </property>
             <property name="singleStep">
              <double>0.001000000000000</double>
             </property>
             <property name="value">
              <double>0.006000000000000</double>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="GQIOption_2">
          <layout class="QHBoxLayout" name="GQIOption">
           <item>
            <widget class="QLabel" name="label_7">
             <property name="text">
              <string>GQI Diffusion sampling ratio</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDoubleSpinBox" name="diffusion_sampling">
             <property name="maximumSize">
              <size>
               <width>75</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="decimals">
              <number>2</number>
             </property>
             <property name="minimum">
              <double>0.200000000000000</double>
             </property>
             <property name="maximum">
              <double>3.000000000000000</double>
             </property>
             <property name="singleStep">
              <double>0.050000000000000</double>
             </property>
             <property name="value">
              <double>1.250000000000000</double>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="ODFDef">
             <property name="maximumSize">
              <size>
               <width>150</width>
               <height>16777215</height>
              </size>
             </property>
             <item>
              <property name="text">
               <string>No distance weighting</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>r-squared weighted</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="OptionGroupBox">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="title">
           <string>Options</string>
          </property>
          <layout class="QGridLayout" name="gridLayout">
           <item row="1" column="0">
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <widget class="QLabel" name="label_2">
               <property name="text">
                <string>ODF Tessellation</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="ODFDim">
               <property name="maximumSize">
                <size>
                 <width>75</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="currentIndex">
                <number>3</number>
               </property>
               <item>
                <property name="text">
                 <string>4-fold</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>5-fold</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>6-fold</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>8-fold</string>
                </property>
               </item>
              </widget>
             </item>
            </layout>
           </item>
           <item row="5" column="0">
            <layout class="QHBoxLayout" name="horizontalLayout_12">
             <property name="topMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QLabel" name="label">
               <property name="text">
                <string>Multithread reconstruction</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="ThreadCount">
               <property name="maximumSize">
                <size>
                 <width>75</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="currentIndex">
                <number>3</number>
               </property>
               <item>
                <property name="text">
                 <string>Off</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>2 threads</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>3 threads</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>4 threads</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>5 threads</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>6 threads</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>7 threads</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>8 threads</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>9 threads</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>10 threads</string>
                </property>
               </item>
              </widget>
             </item>
            </layout>
           </item>
           <item row="7" column="0">
            <widget class="QCheckBox" name="RecordODF">
             <property name="text">
              <string>Record ODF in the output file</string>
             </property>
            </widget>
           </item>
           <item row="7" column="1">
            <widget class="QCheckBox" name="HalfSphere">
             <property name="text">
              <string>Half-sphere scheme</string>
             </property>
             <property name="checked">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <layout class="QHBoxLayout" name="horizontalLayout_6">
             <item>
              <widget class="QLabel" name="label_3">
               <property name="text">
                <string>Number of fibers resolved</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="NumOfFibers">
               <property name="maximumSize">
                <size>
                 <width>75</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="currentIndex">
                <number>2</number>
               </property>
               <item>
                <property name="text">
                 <string>3</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>4</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>5</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>6</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>7</string>
                </property>
               </item>
              </widget>
             </item>
            </layout>
           </item>
           <item row="2" column="1">
            <layout class="QHBoxLayout" name="horizontalLayout_5">
             <property name="spacing">
              <number>0</number>
             </property>
             <item>
              <widget class="QLabel" name="label_4">
               <property name="text">
                <string>Deconvolution</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="ODFSharpening">
               <property name="maximumSize">
                <size>
                 <width>75</width>
                 <height>16777215</height>
                </size>
               </property>
               <item>
                <property name="text">
                 <string>Off</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>On</string>
                </property>
               </item>
              </widget>
             </item>
             <item>
              <widget class="QDoubleSpinBox" name="SharpeningParam">
               <property name="maximumSize">
                <size>
                 <width>75</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="minimum">
                <double>0.500000000000000</double>
               </property>
               <property name="maximum">
                <double>20.000000000000000</double>
               </property>
               <property name="singleStep">
                <double>0.500000000000000</double>
               </property>
               <property name="value">
                <double>3.000000000000000</double>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="2" column="0">
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <property name="topMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QLabel" name="label_9">
               <property name="text">
                <string>ODF Decomposition</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="Decomposition">
               <property name="maximumSize">
                <size>
                 <width>75</width>
                 <height>16777215</height>
                </size>
               </property>
               <item>
                <property name="text">
                 <string>Off</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>On</string>
                </property>
               </item>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QCommandLinkButton" name="doDTI">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>50</height>
             </size>
            </property>
            <property name="text">
             <string>Run reconstruction</string>
            </property>
            <property name="description">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
